tox.ini

```diff
@@ -28,8 +28,8 @@ setenv =
     # happen inside docker, so we use docker names here
     H_US_DATABASE_URL = {env:H_US_DATABASE_URL:postgresql://report-fdw:password@h_postgres_1:5432/postgres}
     H_CA_DATABASE_URL = {env:H_CA_DATABASE_URL:postgresql://report-fdw:password@h_postgres_1:5432/postgres}
-    LMS_US_DATABASE_URL = {env:LMS_US_DATABASE_URL:postgresql://report-fdw:password@lms_postgres_1:5432/postgres}
-    LMS_CA_DATABASE_URL = {env:LMS_CA_DATABASE_URL:postgresql://report-fdw:password@lms_postgres_1:5432/postgres}
+    LMS_US_DATABASE_URL = {env:LMS_US_DATABASE_URL:postgresql://report-fdw:password@lms-postgres-1:5432/postgres}
```

Comment:
Mmm locally I have `lms-postgres-1` but `h_postgres_1`

:thinking:

Reply:
Yep, I think this is because LMS has had the cookie-cutter template rolled out to it, and H hasn't.